recipes/jenkspy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "jenkspy" %}
+{% set version = "0.1.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6bbd8a104b65d76443e6609478d581b9d14bfab42c75c43aaf4a015e845418cf" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
It looks like this is build a C-extension, so could you add `toolchain` to the `build` requirements.